### ISSUE TITLE
window.load event not getting fired when I open a link in a new tab

### ISSUE THREAD

**Issue by sanjay-github** [jquery/jquery#3911]
I am opening a link in new tab using **Ctrl+click**. Simultaneously I am opening 6 to 7 tabs. 

On few pages I am getting it and on some its not working.

**Browser - Chromium ,Version 53.0.2785.143 Built on Ubuntu , running on LinuxMint 17.3 (64-bit)**

This is the my code.

`$(window).load(function () {       alert('here'); });`

What could be the possible issue ?

I have even tried this but no luck.

`document.addEventListener('readystatechange', function(){    if(document.readyState == "complete"){       console.log(document.readyState);    } });`

Basically I want to fire an event when whole page is completely loaded.
--

**Comment by gibson042** [comment]
The problem with binding to one-time events is that if registration happens _after_ they fire, the listener is never invoked.

You're looking for [`jQuery( callback )`](http://api.jquery.com/jQuery/#jQuery3), which invokes callbacks even if the document is already loaded.

**Comment by sanjay-github** [comment]
That would not give me a way to fire an event on whole page load.

```
jQuery(function( $ ) {
  alert('Page completely loaded');
});
```

**Comment by dmethvin** [comment]
This is more of a StackOverflow thing, it's certainly not a jQuery bug report because our APIs are working as they always have. Take a look at `document.readyState` and followup to StackOverflow if you have other needs.
